src/commands/stop.ts

```diff
@@ -0,0 +1,56 @@
+import chalk from 'chalk';
+import BaseCommand from "../base-command";
+import { DockerComposeUtils } from "../common/docker-compose";
+import inquirer from 'inquirer';
+import { DockerInspect } from '../common/docker-compose/template';
+
+export default class StopLocalDeployment extends BaseCommand {
+  async auth_required(): Promise<boolean> {
+    return false;
+  }
+
+  static description = 'Stop a local deployment';
+  static examples = [
+    'architect stop <local-environment-name>',
+  ];
+
+  static flags = {
+    ...BaseCommand.flags,
+  };
+
+  static args = [{
+    sensitive: false,
+    name: 'name',
+    description: 'Name of local environment',
+    required: false,
+  }];
+
+  async run(): Promise<void> {
```

Comment:
This should have the `@RequiresDocker({ compose: true })` decorator on it - I just now realized I (already) forgot about this and didn't include it in the dev:list command (see https://github.com/architect-team/architect-cli/pull/708 - I just now fixed it 😄)